concourse/steps/update_component_deps.py

```diff
@@ -377,6 +382,163 @@ def _import_release_notes(
     return release_notes
 
 
+def format_bom_diff(
+    component_diff: cnudie.util.ComponentDiff,
+    to_component_name: str,
+    base_url: str | None
```

Comment:
`base_url` is not a good name, as it gives no indication as to which kind of URL is expected here. rather call it `delivery_dashboard_url`. It is also not that nice to hard-code assumptions about delivery-dashboard's query-parameters (later in code), but that's okay-ish as long as we do not have a better approach (e.g. via delivery-service).